docs/getting_started.md

```diff
@@ -99,22 +99,21 @@ make the dependencies available.
 
 If you have access to a machine running [CVMFS](https://cernvm.cern.ch/fs/),
 e.g. CERNs lxplus login machines, the dependencies can be easily satisfied
-via a LCG releases available through CVMFS. A setup script is provided to activate a compatible releases that can be used as follows:
+via a LCG releases available through CVMFS. Source the cvmfs setup script
+provided by the machine. It is suggested to select a recent `<lcg_release>`
+and `<lcg_platform>` combination. (Have a look at the CI jobs to get an
+overview on what we are currently testing):
 
 ```console
-cd <source>
-source CI/setup_cvmfs_lcg.sh
+source /cvmfs/sft.cern.ch/lcg/views/<lcg_release>/<lcg_platform>/setup.sh
 ```
 
-After sourcing the setup script, you can build ACTS as described above. The
-following commands will build ACTS in the `<source>/build` directory with the
-Fatras component.
+After sourcing the setup script, you can build ACTS as described above.
+Recall that you can set the necessary dependency variables for your current
+LCG with the pattern:
 
 ```console
-cd <source>
-source CI/setup_cvmfs_lcg.sh
-cmake -B build -S . -DACTS_BUILD_FATRAS=on
-cmake --build build
+export DD4hep_DIR=/cvmfs/sft.cern.ch/lcg/views/<lcg_release>/<lcg_platform>
```

Comment:
I **believe** our generated `build/this_acts_withdeps.sh` handles this correctly. Maybe we should just refer to that?

Reply:
Oh yes, I thought we need it already for cmake, but that should be then handled by CVMFS already.

I think we are missing a general mentioning of our `this_acts*.sh`-scripts. Let me think of a good spot. But that's a different PR, I think.

Reply:
we use the script in the CI so there is some coverage

https://github.com/acts-project/acts/blob/a7261530153e653878ed7f3147c6926451cf6fd1/.github/workflows/builds.yml#L159C24-L159C42

Reply:
Yes, but not the `setup_cvmfs_lcg*`-scripts.

```
$ grep -IRn "setup_cvmfs_lcg"
./CI/setup_cvmfs_lcg.sh:14:source $dir/setup_cvmfs_lcg105.sh
./docs/getting_started.md:106:source CI/setup_cvmfs_lcg.sh
./docs/getting_started.md:115:source CI/setup_cvmfs_lcg.sh
```